x-pack/plugins/infra/common/inventory_models/shared/components/metrics_and_groupby_toolbar_items.tsx

```diff
@@ -8,6 +8,7 @@ import React, { useMemo } from 'react';
 import { EuiFlexItem } from '@elastic/eui';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { WaffleSortControls } from '../../../../public/pages/metrics/inventory_view/components/waffle/waffle_sort_controls';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
@elastic/logs-metrics-ui I've added this ignore rule here as it was already being done for other similar cases across this file. Ideally we really want to avoid this kind of imports with code from public or server code into common code, so it would be really nice if you could remove the eslint ignore rules and instead import the types from the common folder in the future. For now I think we can proceed using the eslint ignore rule 😃

Reply:
Instead of using disable-next-line, can we add it to the rule and create an issue for the team to resolve? https://github.com/elastic/kibana/pull/67149/files#diff-e4403a877d80de653400d88d85e4801aR203

Reply:
@tylersmalley other `disable-next-line` rules were already being applied on that file. Do you think that instead of disable the rule entirely in the infra plugin we can just use the `disable-next-line` and still open the issue to solve this? I think by not turning of the rule we can at least avoid future propagation of similar wrong imports in the future. What do you think?

Reply:
Issue opened https://github.com/elastic/kibana/issues/67239